include/rocksdb/statistics.h

```diff
@@ -204,6 +204,9 @@ enum Tickers : uint32_t {
   COMPACT_READ_BYTES,   // Bytes read during compaction
   COMPACT_WRITE_BYTES,  // Bytes written during compaction
   FLUSH_WRITE_BYTES,    // Bytes written during flush
+  MEMTABLE_DATA_BYTES,  // Raw bytes of data present on memtable at flush time.
```

Comment:
We should also be planning for a (possible / likely?) future when flush could go to the next memtable rather than to storage. How do these tickers fit into that picture, and are the names appropriate?

Reply:
A possible change in terminology: a flush to the next emmtable could be called "a purge" (as we are purging the garbage out of the memtable). The tickers have been renamed MEMTABLE_PAYLOAD_BYTES_AT_FLUSH and MEMTABLE_GARBAGE_BYTES_AT_FLUSH respectively (names still subject to changes). In the future, these statistics will not be incremented at purge time, only at flush time, to better reflect the impact of the purging policy on the ratio of garbage in the memtable.